### ISSUE TITLE
Import external fio data to generate chart

### ISSUE THREAD

**Issue by marcelogigliotti** [pcuzner/fioloadgen#107]
Is there a way to import a kind of external fio result to generate the chart?

**Comment by pcuzner** [comment]
No. The API enables profiles to be loaded, but not job results. I don't have any plans for that.
